### PR TITLE
Removing the time interval check and throwing an error message for de…

### PR DESCRIPTION
**Pull request by goyalvarun601**
…sktops as well.

@kdimatteo

### CODE REVIEW

File: dist/jquery.audio-plugin.js [hunk 1]
```diff
@@ -83,13 +68,10 @@
...
 
-  MHEAudio.prototype.canplay = function(){
-    this.canPlay = true;
-  };
-
   MHEAudio.prototype.errorHandler = function(){
     this.inError = true;
-    if(this.isTablet && this.isPlaying) {
+    if(this.isPlaying) {
+      this.song.src = 'http://destroyloop.com/sessions/intro1.mp3';
```
Comment (kdimatteo) [inline dist/jquery.audio-plugin.js]: :bomb: :boom:

Comment (goyalvarun601) [inline dist/jquery.audio-plugin.js]: oops.

**Comment by goyalvarun601** [comment]
Looks good to go?

**Comment by kdimatteo** [comment]
:+1: :+1:

**Pull request merged**